lib/human_console_player.rb

```diff
@@ -7,6 +7,10 @@ def initialize(mark, ui)
     @ui = ui
   end
 
+  def ready?
```

Comment:
Thanks Georgina! I've already got a yellow mark for duplication in my "table of improvements". Glad you caught this one ahead of my IPM :wink: